app/views/documents/show/_history_tab.html.erb

```diff
@@ -11,6 +11,17 @@
       <%= t "documents.history.entry_types.#{entry.entry_type}" %>
     </h4>
 
+    <% if entry.retire.present? %>
+      <%= entry.retire.explanatory_note %>
+    <% elsif entry.remove.present? %>
```

Comment:
Small thing, but is the else needed here? I'm reading it as "either we'll have a remove, or we'll have a retire", but the .present? check then makes that redundant.

Reply:
You could have neither.